
apriltag_ros_single_image_server:
  ros__parameters:
    standalone_tags:
      tag1:
        id: 1
        size: 20.0
      tag2:
        id: 2
        size: 22.0
        name: Name2
        extra: 65
    tag_bundles:
      bundle1:
        name: "CUSTOM_BUNDLE_NAME"
        layout:
          tag1:
            id: 1
            size: 20.0
            x: 30.0
            y: 31.0
            z: 32.0
            qw: 1
            qx: 0
            qy: 0
            qz: 0
          tag2:
            id: 2
            size: 22.0
            x: 33.0
            y: 34.0
            z: 35.0
            qw: 2
            qx: 0
            qy: 0
            qz: 0
      bundle2:
        name: "CUSTOM_BUNDLE_NAME2"
        layout:
          tag2:
            id: 3
            size: 27.0
            x: 36.0
            y: 37.0
            z: 38.0
            qw: 3
            qx: 0
            qy: 0
            qz: 0
